### PR TITLE
Check conformance to grpc.GenericRpcHandler type

### PR DESCRIPTION
**Pull request by nathanielmanistaatgoogle**

### CODE REVIEW

File: src/python/grpcio/grpc/_server.py [hunk 1]
```diff
@@ -817,3 +827,10 @@ def stop(self, grace):
 
     def __del__(self):
         _stop(self._state, None)
+
+
+def create_server(thread_pool, generic_rpc_handlers, interceptors, options,
```
Comment (mehrdada) [inline src/python/grpcio/grpc/_server.py]: Suggestion: the arguments to this function are numerous and hard to remember. How about naming all of them?

File: src/python/grpcio_tests/tests/unit/_server_test.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 gRPC authors.
```
Comment (mehrdada) [inline src/python/grpcio_tests/tests/unit/_server_test.py]: `s/gRPC authors./The gRPC Authors/` for new files

File: src/python/grpcio/grpc/_server.py (hunk 1 above)
Comment (nathanielmanistaatgoogle) [reply src/python/grpcio/grpc/_server.py]: ... for the moment I still like following the practice "always pass positional arguments positionally and keyword arguments by keyword".

But you do have a point. Maybe in the future there will be a role to play for a `ServerParameters` object or something?

File: src/python/grpcio_tests/tests/unit/_server_test.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 The gRPC Authors.
```
Comment (nathanielmanistaatgoogle) [inline src/python/grpcio_tests/tests/unit/_server_test.py]: Can I keep the trailing punctuation? I've always liked the trailing punctuation.

Comment (mehrdada) [reply src/python/grpcio_tests/tests/unit/_server_test.py]: You can, according to the regex, but I think the way I've seen most people use it, including grpc-Java, Kubernetes, and the Apache license template itself is without punctuation. It's more like a header than a sentence IMO.

**Pull request merged**